### ISSUE TITLE
Teaching Excellence Award

### ISSUE THREAD

**Issue by Gudahtt** [MUNComputerScienceSociety/MUNCS-Website#61]
This this award still a thing? It just occurred to me that the website doesn't list anything about it.

It might be nice to have a page for it, with a list of the past winners.

**Comment by nint8835** [comment]
We didn't do it this past year, but if you could point me to a list of past winners I could do up a list of past winners and bring up restarting it at the next meeting

**Comment by Gudahtt** [comment]
Well it was just an idea - no pressure or anything. I just thought of it because somebody asked me about the awards, and I didn't know where to look for past winners.

Here are the ones I found in my email:

* Winter 2018: Dr. Churchill and Dr. Wareham
* Winter 2016: (Not sure, but I know we did it because I found mention of it)
* Winter 2015: Dr. Antonina Kolokolova
* Fall 2014: Dr. Rod Byrne

Not sure about 2017 either. There might be some record of it in the society gmail account.
